### DISCUSSION TITLE
How to add images without an object into training set? So blank images so the model will learn that there is no such objects..

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [ultralytics/yolov5#10004] [Q&A]
How to add images without an object into training set? So blank images so the model will learn that there is no such objects..

**Comment by spacewalk01** [comment]
Such images can be added into image folder in training/valid set without label file. No labels are required for background/blank images. https://github.com/ultralytics/yolov5/wiki/Tips-for-Best-Training-Results
